quesma/queryparser/aggregation_parser_test.go

```diff
@@ -723,6 +723,35 @@ func sortAggregations(aggregations []*model.Query) {
 	})
 }
 
+func allAggregationTests() []testdata.AggregationTestCase {
+	const lowerBoundTestNr = 80
+	allTests := make([]testdata.AggregationTestCase, 0, lowerBoundTestNr)
+	allTests = append(allTests, testdata.AggregationTests...)
+	allTests = append(allTests, testdata.AggregationTests2...)
+	allTests = append(allTests, opensearch_visualize.AggregationTests...)
+	allTests = append(allTests, dashboard_1.AggregationTests...)
+	allTests = append(allTests, testdata.PipelineAggregationTests...)
+	allTests = append(allTests, opensearch_visualize.PipelineAggregationTests...)
+	allTests = append(allTests, kibana_visualize.AggregationTests...)
+	allTests = append(allTests, clients.KunkkaTests...)
+	allTests = append(allTests, clients.OpheliaTests...)
+	return allTests
+}
+
+// TODO remove after pipeline aggregations are implemented
+func allAggregationTestsWithoutPipeline() []testdata.AggregationTestCase {
+	const lowerBoundTestNr = 80
```

Comment:
Nit, but we don't need it. It is mere a very minor performance optimization in tests.